api/swagger-batch.yaml

```diff
@@ -0,0 +1,310 @@
+      tags: [ "Ensembler" ]
+      summary: Updates existing Ensembler with the data provided in the payload
+      parameters:
+        - in: path
+          name: project_id
+          schema:
+            $ref: "#/components/schemas/Id"
+          required: true
+        - in: path
+          name: ensembler_id
+          schema:
+            $ref: "#/components/schemas/Id"
+          required: true
+      requestBody:
+        description: A JSON object, representing an update to the ensembler
+        required: true
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - $ref: "#/components/schemas/PyFuncEnsembler"
+      responses:
+        "200":
+          description: A JSON object representing an instance of updated Ensembler
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: "#/components/schemas/PyFuncEnsembler"
+  "/projects/{project_id}/jobs":
+    get:
+      summary: Returns a list of Ensembling jobs
+      responses:
+        "200":
+          description: TBU
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/EnsemblingJob"
+
+components:
+  schemas:
+    Id:
+      type: "integer"
+      format: "int32"
+    EnsemblerType:
+      type: "string"
+      enum:
+        - pyfunc
+    GenericEnsembler:
+      properties:
+        id:
+          $ref: "#/components/schemas/Id"
+        project_id:
+          $ref: "#/components/schemas/Id"
+        name:
+          type: "string"
+          minLength: 3
+          maxLength: 50
+        type:
+          $ref: "#/components/schemas/EnsemblerType"
+        created_at:
+          type: "string"
+          format: "date-time"
+        updated_at:
+          type: "string"
+          format: "date-time"
+    PyFuncEnsembler:
+      allOf:
+        - $ref: "#/components/schemas/GenericEnsembler"
+        - type: "object"
+          properties:
+            mlflow_url:
+              type: "string"
+              format: "uri"
+            mlflow_experiment_id:
+              $ref: "#/components/schemas/Id"
+            mlflow_run_id:
+              type: "string"
+            artifact_uri:
+              type: "string"
+              format: "uri"
+    EnsemblingJob:
+      description: A JSON object that represents an ensembling job for batch experiment use cases
+      properties:
+        id:
+          $ref: "#/components/schemas/Id"
+        project_id:
+          $ref: "#/components/schemas/Id"
+
+    EnsemblingJobMeta:
+      description: Ensembling job Metadata
+      properties:
+        name:
+          type: "string"
+        annotations:
+          type: "object"
+          additionalProperties:
+            type: "string"
+
+    EnsemblingJobSpec:
+      description: Ensembling job Specification
+      properties:
+        source:
```

Comment:
Simply because I haven't added it :) The `jobs` spec is out of scope of this PR, but I just started adding some draft for sharing the context with @ashwinath, this can be extended as the part of his PR

Reply:
Yeah I'll handle this in my PR. There's going to be a lot more than that.